docs/uperf.md

```diff
@@ -44,7 +44,6 @@ spec:
       multus:
         enabled: false
       samples: 1
-      pair: 1
```

Comment:
so, I wonder if we can keep pairs for the simple use case... Similar to the pin/collocate options.... if pin is set, it ignores the other options.

Reply:
OK. Will keep pair and make it higher precedence than 'density_range' for pin mode.